saleor/graphql/shop/utils.py

```diff
@@ -2,6 +2,8 @@
 
 from django_countries import countries
 
+from saleor.site.models import SiteSettings
```

Comment:
For importing from Saleor modules use relative imports for consistency :)

Reply:
Ops silly me :)